src/NuGet.Core/NuGet.Protocol/VisualStudio/ExtensibleSourceRepositoryProvider.cs

```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+#if IS_DESKTOP
```

Comment:
That sounds like a good idea. Let me search and double check, I thought PM used this, but I hadn't verified that.